dtmi/belden/hirschmann_opedge_8d-2.json

```diff
@@ -0,0 +1,23 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:Belden:Hirschmann_OpEdge_8D;2",
+    "@type": "Interface",
+    "displayName": "Belden-Hirschmann-OpEdge-8D",
+    "contents": [
+		{
+		  "@type": "Property",
+		  "name": "version",
+		  "displayName": "Version",
+		  "description": "Version of BHDO.",
+		  "writable": false,
+		  "schema": "double"
+		},
+		{
+		  "@type": "Telemetry",
+		  "name": "value1",
```

Comment:
what is `value1` ? I think a better name would be desirable.

Reply:
Canceling the pull request. Need to revise the model.